solved_exercises/dani/04-functions.py

```diff
@@ -1,20 +1,27 @@
-print("Calculate the factorial of a number (positive integer).")
+#Calculate the factorial of a number (positive integer).
 def factorial (n):
     if n > 0:
         result = n * factorial(n-1)
     else:
         result = 1
     return result
 value = int(input("Please enter a numeric value:\n"))
-print("Factorial of",value,"is",factorial(value))
+print('Factorial of',value,'is',factorial(value))
 
-print("Write a function that flattens a list.")
-def flat (lista):
-	return [val for sublist in lista for val in sublist]
-given_list = [[18.0], [13.8], [16.2], [56.5], [147], [18.2]]
-print (flat(given_list))
+#Write a function that flattens a list.
+flat_list = []
```

Comment:
can you refactor this to get rid of the global variable ? the function should take a list as a parameter and return a new list that is the flattened version

Reply:
yes, new commit is available